File: velolabs_repos/lattis_android/app/src/main/java/com/lattis/ellipse/data/network/model/response/parking/FindParkingResponse.java
package com.lattis.ellipse.data.network.model.response.parking;

import com.google.gson.annotations.SerializedName;
import com.lattis.ellipse.data.network.model.response.AbstractApiResponse;

import java.util.List;

/**
 * Created by ssd3 on 3/21/17.
 */

public class FindParkingResponse extends AbstractApiResponse {

    @SerializedName("payload")
    FindParkingPayloadResponse findParkingPayloadResponse;

    public List<FindParkingDataResponse> getFindParkingResponse() {
        return findParkingPayloadResponse.findParkingDataResponses();
    }

    @Override
    public String toString() {
        return "GetUserResponse{" +
                "FindBikeDataResponse=" + findParkingPayloadResponse +
                '}';
    }
}

File: velolabs_repos/lattis_android_v2/data/src/main/java/com/lattis/data/entity/response/parking/FindParkingResponse.kt
package com.lattis.data.entity.response.parking

import com.google.gson.annotations.SerializedName
import com.lattis.data.entity.response.base.AbstractApiResponse

class FindParkingResponse : AbstractApiResponse() {
    @SerializedName("payload")
    var findParkingPayloadResponse: FindParkingPayloadResponse? = null
    fun findParkingDataResponse(): List<FindParkingDataResponse>?
    {
        if(findParkingPayloadResponse!=null && findParkingPayloadResponse?.findParkingDataResponses!=null){
            return findParkingPayloadResponse!!.findParkingDataResponses()
        }else{
            return null
        }

    }

    override fun toString(): String {
        return "GetUserResponse{" +
                "FindBikeDataResponse=" + findParkingPayloadResponse +
                '}'
    }
}